lib/nextstrain/sphinx/theme/theme.conf

```diff
@@ -8,3 +8,4 @@ pygments_style = default
 style_nav_header_background = #f2f2f2
 logo = True
 logo_link = https://docs.nextstrain.org
+subproject = True
```

Comment:
Do each of these config attributes appear in the template namespace as `theme_{param}`? Just making sure I understand how this maps to `theme_subproject` in the template code.

Reply:
Yep, they walk right in just like that and make themselves at home in the global template context.